### ISSUE TITLE
Please do: Botania relics 

### ISSUE THREAD

**Issue by Jules06** [F32Organization/Faithful32-1.7.10#335]
Thanks !

**Comment by Evolution0** [comment]
I'm guessing its a newer core feature that we haven't textured yet?

**Comment by Jules06** [comment]
No, these are just some recently added item dropped by the new boss

**Comment by Evolution0** [comment]
Seems like Botania has a lot more to be done in general, I generally contribute to tech mods (I don't even play magic mods) but I at least updated the TODO directory to the recent version of Botania.

Gonna wait for my other PR to go through before I make one for it.

@StefanTT Seems to be the main contributor of Botania textures but Id be open to at least help things along (There are a lot from what I can see @_@)
